geode-pulse/src/main/webapp/WEB-INF/web.xml

```diff
@@ -32,7 +32,7 @@
   </servlet>
   <servlet-mapping>
     <servlet-name>mvc-dispatcher</servlet-name>
-    <url-pattern>/pulse/*</url-pattern>
```

Comment:
It still works because of the "/pulse" webAppContext string passed into `JettyHelper.addWebApplication()`. Jetty only passes requests for URLs matching this webAppContext into the Pulse webapp, so if a URL doesn't match /pulse it never makes it into this servlet mapping.  However, maybe we still ought to leave it as `<url-pattern>/pulse/*</url-pattern>` for clarity of intent.